index.html

```diff
@@ -0,0 +1,233 @@
+            <h5 class="pricing__card--cost-intervall">per month</h5>
+            <ul class="pricing__card--feature">
+              <li>Up to 7 Projects</li>
+              <li>2 Additional Developers</li>
+            </ul>
+            <button class="btn pricing__card--btn">Get Started</button>
+          </div>
+        </div>
+        <div class="pricing__container--card">
+          <div class="pricing__card">
+            <h3 class="pricing__card--title">AGENCY</h3>
+            <h4 class="pricing__card--cost">$55</h4>
+            <h5 class="pricing__card--cost-intervall">per month</h5>
+            <ul class="pricing__card--feature">
+              <li>Up to 25 Projects</li>
+              <li>2 Additional Developers</li>
+              <li>Unlimited Support</li>
+            </ul>
+            <button class="btn pricing__card--btn">Get Started</button>
+          </div>
+        </div>
+        <div class="pricing__container--card">
+          <div class="pricing__card">
+            <h3 class="pricing__card--title">PRO</h3>
+            <h4 class="pricing__card--cost">$75</h4>
+            <h5 class="pricing__card--cost-intervall">per month</h5>
+            <ul class="pricing__card--feature">
+              <li>Up to 25 Projects</li>
+              <li>2 Additional Developers</li>
+              <li>Unlimited Support</li>
+              <li>1.5GB Disk Space</li>
+            </ul>
+            <button class="btn pricing__card--btn">Get Started</button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="section section__team team">
+      <div class="team__container--title">
+        <h2 class="team__title">THE TEAM</h2>
+      </div>
+      <div class="team__container--cards">
+        <div class="team__card">
+          <div class="team__card--img">
+            <img src="images/cto.png" alt="Johny B Good" />
+          </div>
+          <div class="team__card--title">CTO</div>
+          <div class="team__card--name">Johny B Good</div>
+          <div class="team__card--role-description">
+            The brains behind the whole operation
+          </div>
+          <div class="team__card--social-media">
+            <object
+              class="team__card--social-media-icon"
+              data="./images/team-twitter.svg"
+            ></object>
+            <object
+              class="team__card--social-media-icon"
+              data="./images/team-facebook.svg"
+            ></object>
+            <object
+              class="team__card--social-media-icon"
+              data="./images/team-google.svg"
+            ></object>
+          </div>
+        </div>
+        <div class="team__card">
+          <div class="team__card--img">
+            <img src="images/ceo.png" alt="Johny B Good" />
+          </div>
+          <div class="team__card--title">CEO</div>
+          <div class="team__card--name">Roll Over Beethoven</div>
+          <div class="team__card--role-description">

```

Comment:
Podobnie jak wcześniej brak Elementu z BEM tj. `team__card team__card--title`
- team__card - ma ogólne cechy dla wszystkich elementów
- team__card--title - ma szczególne cechy dla wszystkich elementów (np. zaznaczenie, że element jest aktywny).

Dlatego tutaj to rozwiązanie nie pasuje.